src/open_samus_returns_rando/specific_patches/static_fixes.py

```diff
@@ -322,52 +323,60 @@ def patch_a1_teleporter_crumbles(editor: PatcherEditor) -> None:
 
 
 def disable_vignettes(editor: PatcherEditor) -> None:
-    scenario_block_sg: dict[str, list[dict[str, typing.Any]]] = {
+    scenario_scene_blocks: dict[str, list[dict[str, typing.Any]]] = {
         # Exterior Alpha Bomb Block
         "s010_area1": [
-            {"block_group": 6, "cc": ["017", "018"], "vignette_models": [2177967912]},
+            {"block_group": 6, "names": ["vignettealpha41"]},
         ],
         # Beam Burst Shot Blocks
         "s030_area3": [
-            {"block_group": 5, "cc": ["036"], "vignette_models": [1239739760, 2986218869]},
+            {"block_group": 5,  "names": ["object090", "vignette_050"]},
         ],
         # Lower Chozo Seal Bomb Blocks
         "s040_area4": [
-            {"block_group": 27, "cc": ["006"], "vignette_models": [46617345, 354988048]}
+            {"block_group": 27, "names": ["s040_area4_111", "sg_vinegtte_003"]}
```

Comment:
A comment would help

Reply:
Eh. There's typos all over. It's not the scenario name.

Reply:
IMO its better to explicitly have "hey, mercury goofed this up here" instead of whoever reads this needing to think about "is this an accidental typo or not"?

Reply:
`# [sic]`

Reply:
Idk i disagree still 🤷